### PR TITLE
#85 Create common components

### PR DESCRIPTION
**Pull request by karyna-kh**

### CODE REVIEW

File: frontend/src/components/common/auto-complete/auto-complete.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { useState } from '../../../hooks/hooks';
```
Comment (Pavel1066) [inline frontend/src/components/common/auto-complete/auto-complete.tsx]: change to ''hooks/hooks'

File: frontend/src/components/profile-settings/steps/first-step.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { StepProps } from './common/step-props';
-import TextField from '../../common/text-field/text-field';
+import { TextField } from '../../common/common';
```
Comment (Pavel1066) [inline frontend/src/components/profile-settings/steps/first-step.tsx]: change to ''components/common/common'

File: frontend/package.json [hunk 3]
```diff
@@ -36,6 +37,7 @@
   "devDependencies": {
     "@types/node": "16.11.19",
     "@types/react": "17.0.38",
+    "@types/react-bootstrap-typeahead": "^5.1.8",
```
Comment (nikita-remeslov) [inline frontend/package.json]: lock lib

File: frontend/src/components/common/auto-complete/styles.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
+@import 'src/assets/scss/utils';
```
Comment (nikita-remeslov) [inline frontend/src/components/common/auto-complete/styles.scss]: Use alias `~assets/scss/utils`

File: frontend/src/components/common/modal/styles.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
+@import 'src/assets/scss/utils';
```
Comment (nikita-remeslov) [inline frontend/src/components/common/modal/styles.scss]: same

File: frontend/src/components/common/modal/styles.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  .modal-footer {
+    align-self: start;
+    padding: 25px 50px;
+    background-color: $gu-white;
+  }
+}
+
+@media (max-width: 576px) {
```
Comment (nikita-remeslov) [inline frontend/src/components/common/modal/styles.scss]: try to use breakpoint from bootstrap

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Overall looks good, fix comments and we can merge it

**Review by nikita-remeslov — APPROVED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**